test/quantization/fx/test_quantize_fx.py

```diff
@@ -4082,6 +4082,57 @@ def forward(self, inputs: torch.Tensor, state: List[torch.Tensor]):
             if n.target == "lstm":
                 self.assertEqual(type(n.args[1]), tuple)
 
+    def test_static_lstm(self):
+        """
+        Test FX static quantization for LSTM.
+        """
+        class MyModel(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.lstm = nn.LSTM(50, 50, 1)
+
+            def forward(self, inputs: torch.Tensor, h0: torch.Tensor, c0: torch.Tensor):
+                return self.lstm(inputs, (h0, c0))
+
+        m = MyModel()
+        qconfig_mapping = get_default_qconfig_mapping()
+        prepare_custom_config = PrepareCustomConfig() \
```

Comment:
nit: maybe we can have a get_default_prepare_custom_config as well (can be in a separate PR)